mkdocs.yml

```diff
@@ -0,0 +1,107 @@
+site_name: Aeternity JavaScript SDK
+extra_css: [extra.css]
+theme:
+    name: readthedocs
+    highlightjs: true
+    hljs_languages:
+        - yaml
+        - rust
+        - erlang
+        - javascript
+        - js
+        
+markdown_extensions:
+    - attr_list
+    - admonition
+    - codehilite
+    #- pymdownx.details
+    #- pymdownx.superfences
```

Comment:
```suggestion
```
🤷‍♀️

Reply:
could we leave that in there for now, as those seem to be useful plugins I just can't get to run yet because of whatever bugs/issues currently exist somewhere in mkdocs?

Reply:
Let's make an issue after merging this PR.

Reply:
I have briefly checked all of these plugins and as I understand they implement the support of extra syntax that is not currently used by the docs. I think we should introduce it on demand.